### PR TITLE
add fixture data for post list

### PR DESCRIPTION
**Pull request by forrestbthomas**
Add fixture data for posts

### CODE REVIEW

File: js/components/posts_index.react.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+var Fixtures = require('../fixtures/post-data');
 
 var PostsIndex = React.createClass({
   render: function() {
     return (
+    <div>
       <h1>hi i am a posts index!</h1>
+      <ul>
+        {Fixtures.map(function(item) {
+        return <li>{item}</li>
```
Comment (rorysaur) [inline js/components/posts_index.react.js]: - indent line 11
- make each link an `<a>` element with the item's title as text and the item's URL as href

File: js/fixtures/post-data.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var data = [
+    {
+      "title": "A post",
+      "link": "with/a/link.com",
```
Comment (rorysaur) [inline js/fixtures/post-data.js]: let's call this "url" instead of "link" to be more specific

File: js/components/posts_index.react.js (hunk 1 above)
Comment (forrestbthomas) [inline js/components/posts_index.react.js]: Super weird that the indenting didn't show up, because it's indented on my local.

Comment (rorysaur) [inline js/components/posts_index.react.js]: :hankey:

Comment (forrestbthomas) [inline js/components/posts_index.react.js]: Did you just poop on my comment to your comment?

Rude...

**Pull request merged**